### PR TITLE
feat(APIAuditLogChange): add missing keys

### PR DESCRIPTION
**Pull request by fiftyy**
**Please describe the changes this PR makes and why it should be merged:**
Missing keys, found it while doing some stuff with forum channels.
```ts
		const auditLogEntry = (await newChannel.guild.fetchAuditLogs({ type: AuditLogEvent.ChannelUpdate, limit: 1 })).entries.first();
		auditLogEntry.changes.forEach((change) => {
			console.log(change.key + " -> " + JSON.stringify(change.new))
        });
```

```
flags -> 16
available_tags -> [{"id":"1227019093945618513","name":"g","emoji_id":"1171597283180163152","emoji_name":null,"moderated":false},{"id":"1227019128099962970","name":"Refund","emoji_id":null,"emoji_name":"💯","moderated":false},{"id":"1227019436351819937","name":"Question","emoji_id":null,"emoji_name":"❔","moderated":false},{"id":"1227019287827578952","name":"Priority: High","emoji_id":null,"emoji_name":"⚠️","moderated":false},{"id":"1232485370864865300","name":"fggfgdhj","emoji_id":"999745369527308358","emoji_name":null,"moderated":false}]
default_reaction_emoji -> undefined
default_reaction_emoji -> {"emoji_id":"986427353230098543","emoji_name":null}
default_thread_rate_limit_per_user -> 30
```

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**

### CODE REVIEW

File: deno/payloads/v10/auditLog.ts [hunk 1]
```diff
@@ -826,6 +830,27 @@ export type APIAuditLogChangeKeyExemptRoles = AuditLogChangeData<'exempt_roles',
  */
 export type APIAuditLogChangeKeyExemptChannels = AuditLogChangeData<'exempt_channels', Snowflake[]>;
 
+/**
+  * Returned when a guild forum's available tags gets changed
+ */
+export type APIAuditLogChangeKeyAvailableTags = AuditLogChangeData<'available_tags', APIGuildForumTag[]>
```
Comment (vladfrangu) [inline deno/payloads/v10/auditLog.ts:836]: You need to add these in the payloads/v10, payloads/v9, then run the scripts and it'll update the deno files
Also linting

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Review by vladfrangu — APPROVED** [review]

File: deno/payloads/v10/auditLog.ts [hunk 2]
```diff
@@ -826,6 +830,33 @@ export type APIAuditLogChangeKeyExemptRoles = AuditLogChangeData<'exempt_roles',
...
+	'default_reaction_emoji',
+	APIGuildForumDefaultReactionEmoji
+>;
+
+/**
+ * Returned when a channel flag gets changed
+ */
+export type APIAuditLogChangeKeyFlags = AuditLogChangeData<'flags', number>;
+
+/**
```
Comment (vladfrangu) [inline deno/payloads/v10/auditLog.ts:851]: This is a bit of a copy paste fail, it relates to making threads, not sending messages

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: deno/payloads/v10/auditLog.ts (hunk 2 above)
Comment (fiftyy) [reply deno/payloads/v10/auditLog.ts:851]: Ah okay, how should we even name this. It only comes when changing this option on a guild forum.
image

Comment (Jiralite) [reply deno/payloads/v10/auditLog.ts:851]: I am pretty sure @vladfrangu is talking about the `APIAuditLogChangeKeyDefaultThreadRateLimitPerUser` below. The description doesn't make sense.

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**